echidna/output/store.py

```diff
@@ -234,20 +234,22 @@ def dump_fit_results(file_path, fit_results, append=False,
         group = file_.create_group(group_name)
         group.attrs["name"] = fit_results._name
         group.attrs["spectra_config"] = json.dumps(
-            fit_results._spectra_config.dump())
+            fit_results._spectra_config.dump(), sort_keys=True)
```

Comment:
I'm don't think we want `sort_keys=True`. The parameters in all configs are now stored in `OrderedDict`s so we always maintain the parameter orderings. Using `sort_keys=True` will just sort the keys alphabetically when writing to file, so that when we load the object again, the order may not be retained.